extensions/positron-python/python_files/positron/positron_ipykernel/data_explorer_comm.py

```diff
@@ -325,8 +332,13 @@ class TableData(BaseModel):
         description="The columns of data",
     )
 
-    row_labels: Optional[List[List[StrictStr]]] = Field(
-        default=None,
+
+class TableRowLabels(BaseModel):
+    """
+    Formatted table row labels formatted as strings
+    """
+
+    row_labels: List[List[StrictStr]] = Field(
```

Comment:
Do we really this to be a list of lists of row labels? I'd expect it to be `List[StrictStr]`, an array of strings for the selected region.

Reply:
pandas has MultiIndexes where you can use multiple layers of row labels, so this is future proofing for when the front end can handle that properly